soundio/soundio.h

```diff
@@ -18,14 +18,18 @@
 #define SOUNDIO_EXTERN_C
 #endif
 
-#if defined(_WIN32)
-#if defined(SOUNDIO_BUILDING_LIBRARY)
-#define SOUNDIO_EXPORT SOUNDIO_EXTERN_C __declspec(dllexport)
+#if defined(SOUNDIO_STATIC_LIBRARY)
+# define SOUNDIO_EXPORT
```

Comment:
Upon further investigation, I think the extern c is needed, even when building a static library. The static files will have symbol names associated with blocks of code, and you want the symbol names to be unmangled, both when creating the static library and when compiling against the static library.